### ISSUE TITLE
FetchError: invalid json response body at https://appleid.apple.com/auth/keys reason: Unexpected token < in JSON at position 0.

### ISSUE THREAD

**Issue by ryanliszewski** [A-Tokyo/apple-signin-auth#19]
Sometimes, I'm getting this error when trying fetch the public apple keys https://appleid.apple.com/auth/keys to verify a users token signature for Apple Signin:

```
FetchError: invalid json response body at https://appleid.apple.com/auth/keys reason: Unexpected token < in JSON at position 0." 
```

Reached out to apple directly too see if there could be an issue on there side of why valid JSON is not being sent back. https://developer.apple.com/forums/thread/656132. In [Apple's docs](https://developer.apple.com/documentation/sign_in_with_apple/fetch_apple_s_public_key_for_verifying_token_signature), they specify to set the content type as so `Content-Type: application/json`. 

One suggestion would be to add a header when fetching for the public keys. I tried pushing a branch but wasn't allow because of access rights. Happy to throw a PR up for this and also help maintain this library. First step would be too convert everything to typescript. 
 
```
onst options = {
    headers: {
      'Content-Type': 'application/json',
    },
    method: 'GET',
  }

  // Fetch Apple's Public keys
  const data = await fetch(url.toString(), options)
    .then((res) => res.json());
```

**Comment by A-Tokyo** [comment]
Hey @ryanliszewski!

Thanks for bringing this up, I fixed and pushed already @1.0.7.

I think the way to open a PR is to fork the repo and THEN create a PR! Would really appreciate it

Keep in mind tho that we still need to support flow, is there a way to support both flow and typescript other than flow-typed?

**Comment by A-Tokyo** [comment]
We can further discuss typescript proposals here https://github.com/A-Tokyo/apple-signin-auth/issues/13 💪
